common/genes/05_genes_special_accessories_clothes.txt

```diff
@@ -1361,6 +1361,18 @@ special_genes = {
 				boy = male
```

Comment:
If you take a close look, you can see a duplicate mesh. I guess it's mirroring.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120005885-605e2580-bfe9-11eb-9c98-677b6fe868f5.png)

</details>